control-plane-operator/controllers/hostedcontrolplane/testdata/kube-controller-manager/zz_fixture_TestControlPlaneComponents.yaml

```diff
@@ -0,0 +1,296 @@
+        - --feature-gates=TranslateStreamCloseWebsocketRequests=false
+        - --feature-gates=VolumeGroupSnapshot=false
+        command:
+        - hyperkube
+        - kube-controller-manager
+        image: hyperkube
+        imagePullPolicy: IfNotPresent
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: healthz
+            port: 10257
+            scheme: HTTPS
+          initialDelaySeconds: 45
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 10
+        name: kube-controller-manager
+        ports:
+        - containerPort: 10257
+          name: client
+          protocol: TCP
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: healthz
+            port: 10257
+            scheme: HTTPS
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 10
+        resources:
+          requests:
+            cpu: 60m
+            memory: 400Mi
+        volumeMounts:
+        - mountPath: /var/run/kubernetes
+          name: certs
+        - mountPath: /etc/kubernetes/certs/cluster-signer
+          name: cluster-signer
+        - mountPath: /etc/kubernetes/config
+          name: kcm-config
+        - mountPath: /etc/kubernetes/secrets/svc-kubeconfig
+          name: kubeconfig
+        - mountPath: /var/log/kube-controller-manager
+          name: logs
+        - mountPath: /etc/kubernetes/recycler-config
+          name: recycler-config
+        - mountPath: /etc/kubernetes/certs/root-ca
+          name: root-ca
+        - mountPath: /etc/kubernetes/certs/server
+          name: server-crt
+        - mountPath: /etc/kubernetes/certs/service-signer
+          name: service-signer
+      initContainers:
+      - command:
+        - /usr/bin/control-plane-operator
+        - availability-prober
+        - --target
+        - https://kube-apiserver:6443/readyz
+        image: availability-prober
+        imagePullPolicy: IfNotPresent
+        name: availability-prober
+        resources: {}
+      priorityClassName: hypershift-control-plane
+      restartPolicy: Always
+      schedulerName: default-scheduler
+      tolerations:
+      - effect: NoSchedule
+        key: hypershift.openshift.io/control-plane
+        operator: Equal
+        value: "true"
+      - effect: NoSchedule
+        key: hypershift.openshift.io/cluster
+        operator: Equal
+        value: hcp-namespace
+      volumes:
+      - configMap:
+          defaultMode: 420
+          name: kcm-config
+        name: kcm-config
+      - configMap:
+          defaultMode: 420
+          name: root-ca
+        name: root-ca
+      - emptyDir: {}
+        name: logs
+      - name: kubeconfig
+        secret:
+          defaultMode: 416
+          secretName: kube-controller-manager-kubeconfig
+      - name: cluster-signer
+        secret:
+          defaultMode: 416
+          secretName: cluster-signer-ca
```

Comment:
tangential to the PR but why is testdata outside v2/?

Reply:
I see it's probably to satisfy CompareWithFixture from TestControlPlaneComponents

Reply:
right, CompareWithFixture generates testdata for `TestControlPlaneComponents` there, since the test is defined in hostedcontrolplane_controller_test.go